app/models/invenio_rdm_record_converter.rb

```diff
@@ -0,0 +1,48 @@
+include Sufia::Export
+
+# Convert a GenericFile including metadata, permissions and version metadata into a PORO
+# so that the metadata can be exported in json format using to_json
+#
+class InvenioRdmRecordConverter < Sufia::Export::Converter
+  # Create an instance of a InvenioRdmRecordConverter converter containing all the metadata for json export
+  #
+  # @param [GenericFile] generic_file file to be converted for export
+  def initialize(generic_file)
```

Comment:
same here about 'initialize'?

Reply:
Same as above. Just trying to keep thins where it will be easy to reference.